### PR TITLE
Add download script

### PR DESCRIPTION
**Pull request by swarajrao**
Adding a script that returns live progress of s3 downloads
Screen Shot 2022-01-19 at 2 42 19 PM

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - object:
+    opts:
+      title: S3 Object
+      summary: S3 Object to download
+      description: S3 Object to download
+      is_required: true
+      is_sensitive: false
+  - filename:
+    opts:
+      title: S3 Filename
```
Comment (xbrett) [inline step.yml:58]: Is this the S3 filename or the name of the file that will be created locally after downloading the object from S3?

Comment (swarajrao) [reply step.yml:58]: The filename that will be created locally after downloading. I'll update the var names to be more descriptive

**Comment by swarajrao** [comment]
example of the new step in action, only took ~25s here: https://app.bitrise.io/build/abcefd28-8c99-4b90-8107-edb4c78fa09a#?tab=log

**Review by xbrett — APPROVED** [review]
Very nice! Huge win with ~2 min off every build

**Pull request merged**